docker/fetch-data/Dockerfile

```diff
@@ -0,0 +1,17 @@
+
+FROM demisto/python3:3.8.3.9324
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
+  && apk add --no-cache python3-dev libstdc++  \
```

Comment:
These should be installed as part of the .build-dependencies so they are removed when the step completes. Same goes for g++.

Reply:
Thanks, when I have it like this:
```
RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git python3-dev libstdc++ g++ \
  && ln -s /usr/include/locale.h /usr/include/xlocale.h \
  && pip3 install --no-cache-dir -r requirements.txt \
  && apk del .build-dependencies
```

I'm having problems importing Pandas 
` ImportError: Error loading shared library libstdc++.so.6: No such file or directory (needed by /usr/local/lib/python3.8/site-packages/pandas/_libs/window/aggregations.cpython-38-x86_64-linux-gnu.so)`

Reply:
I see. So you should have only libstdc++ on a separate "apk add"